### DISCUSSION TITLE
The Repo is 300 MB big?

### DISCUSSION THREAD

**Discussion by Snickbrack** [d2phap/ImageGlass#983] [General] [answered]
Why is the repo for a 40 MB Application so big?

Couldn't you split the repo into different pieces and us the Wiki functionality inside Github?

**Comment by fire-eggs** [comment, marked as answer]
This is part of how Git works. If you downloaded the ZIP you'd find that (unpacked) it was 115M. By performing a deep clone, you are getting over 8 years of development history, and that adds up!

On one point you are correct - the Assets sub folder could conceivably be split out.
